src/main/java/com/facebook/openwifirrm/ucentral/informationelement/Country.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.informationelement;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+/**
+ * This information element (IE) appears in wifiscan entries.
+ * Refer to the 802.11 specification for more details. Language in
+ * javadocs is taken from the specification.
+ */
+public class Country {
+	private static final Logger logger = LoggerFactory.getLogger(Country.class);
+
+	/** Defined in 802.11 */
+	public static final int TYPE = 7;
+
+	public static class CountryInfo {
+		/**
+		 * The lowest channel number in the CountryInfo.
+		 */
+		public final int firstChannelNumber;
+		/**
+		 * The maximum power, in dBm, allowed to be transmitted.
+		 */
+		public final int maximumTransmitPowerLevel;
+		/**
+		 * Number of channels this CountryInfo applies to. E.g., if First
+		 * Channel Number is 2 and Number of Channels is 4, this CountryInfo
+		 * describes channels 2, 3, 4, and 5.
```

Comment:
I don't think channels have to be consecutive numbers

Reply:
Correct, but this is just how the standard describes the limits on the channel. At least, that's what I understand from the "9.4.2.8 Country element" section.